mevolib/cluster/NaiveCols.py

```diff
@@ -1,89 +1,67 @@
-#
-#-------------------------------------------------------------------------------
-
-from __future__ import absolute_import, division
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+"""Clustering where each resulting set is composed by a specific range slice of all the input sequences 
+(column/fragment division)."""
 
 import math
 
-from Bio.SeqRecord import SeqRecord
+from typing import List
 
+from Bio.SeqRecord import SeqRecord
 
-#-------------------------------------------------------------------------------
 
-def map_seqs ( record_list, num_sets ) :
-    """
-    Naive splicing in 'num_sets' sets of the sequences at 'record_list'. The
-    maximum fragment length per set is calculated as follows:
+def map_seqs (record_list: List, num_sets: int) -> dict:
+    """Naive splicing in 'num_sets' sets of the sequences at 'record_list'. The maximum fragment length per 
+    set is calculated as follows:
 
         frag_length_per_set = ceiling( maximum_sequence_length / 'num_sets' )
 
-    Arguments :
-        record_list  ( list )
-            List of SeqRecord objects (from Biopython).
-        seq_length  ( int )
-            Number of sets of sequence slices.
+    Args:
+        record_list: List of SeqRecord objects (from Biopython).
+        seq_length: Number of sets of sequence slices.
 
-    Returns :
-        dict
-            Dictionary with the set identifiers as keys and the corresponding
-            sequence fragments as values in lists of SeqRecord objects.
+    Returns:
+        dict: Dictionary with the set identifiers as keys and the corresponding sequence fragments as values 
+        in lists of SeqRecord objects.
     """
-    # Get maximum and minimum length to determine fragment size for the given
-	# number of sets
-    max_len = len(max(record_list, key=lambda x: len(x.seq)))
+    # Get maximum and minimum length to determine fragment size for the given number of sets
+    max_len = len(max(record_list, key = lambda x: len(x.seq)))
     max_frag_len = int(math.ceil(float(max_len) / num_sets))
-	min_frag_len = max_frag_len - 1
-    # Get the number of sets that will have the maximum length sequences to
-    # balance the distribution per set
+    min_frag_len = max_frag_len - 1
+    # Get the number of sets that will have the maximum length sequences to balance the distribution per set
     big_sets = max_len % num_sets
     small_sets = num_sets - big_sets	
     # Minimum string length for the given number of sets (with zero-filling)
     num_zeros = len(str(num_sets))
     # Generate all the set ids and their corresponding starting site
-	first_range = big_sets * max_frag_len
+    first_range = big_sets * max_frag_len
```

Comment:
The following 5 lines need indentation fixes and update the `.format` for its f-string equivalent.